packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -186,6 +191,15 @@ async function makeE2ETasks () {
       return { launchpadPort }
     },
 
+    /**
+     * Clear out any capability specific environment variables that were set during the test
+     */
+    __internal__afterEach () {
+      delete process.env.CYPRESS_ENABLE_CLOUD_STUDIO
+
+      return null
```

Comment:
Does `null` need to be returned?

Reply:
Yes. You have to return something for these.

Reply:
That's right, these are `cy.task` calls.